internals/parser.rst

```diff
@@ -890,7 +890,7 @@ To activate verbose mode you can add the ``-d`` flag when executing Python:
     $ python -d file_to_test.py
 
 This will print **a lot** of output to ``stderr`` so is probably better to dump it to a file for further analysis. The output
-consists of trace lines with the following structure:
+consists of trace lines with the following structure::
```

Comment:
I tried using `peg` highlighting but it gets confused by the `<...>`.  The default highlight renders nicely so I left that.